packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -91,10 +91,10 @@ export class EntityList extends PureComponent {
       return;
     }
 
-    const { filteredEntities } = this.props;
+    const { entities } = this.props;
```

Comment:
Great rename, I found this rather confusing as well. The EntityList doesn't need to know if the entities are filtered or not!